### PR TITLE
Rest API redis error: wrong number of arguments for 'mget' command

### LINKED ISSUE #8587: Rest API redis error: wrong number of arguments for 'mget' command
### Description

It appears the following redis error happens quite frequently in testnet:

```
2024-06-18T19:40:31.232Z WARN e2b77dc8 Redis error during mget: ERR wrong number of arguments for 'mget' command
2024-06-18T19:40:31.233Z WARN e2b77dc8 x.x.x.x GET /api/v1/accounts/0.0.5897416?&limit=999 in 40 ms: 404 Not found NotFoundError Not found
```

Note the second line should be the request which led to the redis error.

### Steps to reproduce

check the Rest API log

### Additional context

_No response_

### Hedera network

testnet

### Version

v0.107.1

### Operating system

None

### PR DESCRIPTION
**Pull request by jascks**
**Description**:

It appears the following redis error happens quite frequently in `testnet`:

```
2024-06-18T19:40:31.232Z WARN e2b77dc8 Redis error during mget: ERR wrong number of arguments for 'mget' command
2024-06-18T19:40:31.233Z WARN e2b77dc8 x.x.x.x GET /api/v1/accounts/0.0.5897416?&limit=999 in 40 ms: 404 Not found NotFoundError Not found
```

If the keys provided to `mget()`, `mgetBuffer()` etc are empty, then the behavior is to throw this error. There are a few opportunities for `transactions.getTransactionTimestamps()` to return empty rows.  It seems reasonable for `cache.get()` to look for this condition.

**Related issue(s)**:

Fixes #8587

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-rest/cache.js [hunk 1]
```diff
@@ -78,10 +78,13 @@ export class Cache {
       return loader(keys);
     }
 
-    const buffers =
+    const mappedKeys = _.map(keys, keyMapper);
+    const buffers = mappedKeys.length === 0 ? [] :
```
Comment (steven-sheehy) [inline hedera-mirror-rest/cache.js:82]: Would prefer that we revert this and add a fail fast at the top before !ready:

```javascript
    if (_.isEmpty(keys)) {
        return [];
    }
```

Comment (jascks) [reply hedera-mirror-rest/cache.js:82]: Thought maybe `keyMapper` in `_.map()` could delete entries, potentially resulting in an empty array of keys. More digging does not show that being the case. So, reverted and added the suggested fail fast statement.

**Review by steven-sheehy — APPROVED** [review]

**Review by edwin-greene — APPROVED** [review]
Looks good

**Review by mgoelswirlds — APPROVED** [review]
LGTM

**Pull request merged**